### ISSUE TITLE
[Consistency] User encounters design components that are inconsistent or confusing.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80585]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User encounters design components that are inconsistent or confusing.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design), @briandeconinck (Accessibility)

### Description

On the claims page, there are multiple warning alerts stacked on top of each other. Per [our guidance](https://design.va.gov/components/alert#how-to-use-alerts), you should not stack alerts.

## Recommended action

These are notifications, not Alerts, because:
They are triggered not by the user, but by an external process
They should stick around until the user has dealt with them (they are not dismissible but no idea how long they'll stay in this interface) and thus they should have state (read, unread) and the ability to see them later (archive). We would love to see something new proposed for this situtaion however since it is late in the game, choose a hierarchy and show one at a time until it is dealt with. Matt Dingee has filed a design [gap ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2684), if you would like to propose something new please reach out.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Alert ](https://design.va.gov/components/alert)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jacobworrell** [comment]
@shiragoodman noting that the new file upload component we're migrating to does not conform to the standards you gave us as feedback https://github.com/department-of-veterans-affairs/va.gov-team/issues/87835

**Comment by jacobworrell** [comment]
We do not have evidence that the direction given in this feedback helps solves our user's pain points and we have some indication that we'd create brand new ones if we created functionality allowing users to dismiss these to-do items. It's important that the items get dismissed only when the action being requested is taken.  We understand the desire to not make pages too "noisy", but the value of having these high priority items super salient to veterans seems to outweigh this concern. We think the current design approach works for our use cases and would like to close this ticket @shiragoodman. Happy to discuss in future office hours if necessary.

**Comment by shiragoodman** [comment]
@allison0034 and @briandeconinck tagging you both since you flagged this an an issue. See Jacob's comments above and let me know if there's anything we need to escalate. I can pull in Matt if you think it's necessary.

**Comment by pmclaren19** [comment]
@shiragoodman any updates on this? Are we good to close this?

**Comment by shiragoodman** [comment]
@pmclaren19 sorry, this one must've slipped thru the cracks. Let me follow up with me team and I'll get back to you soon.

**Comment by allison0034** [comment]
Ho @pmclaren19, you can work on a solution that does not stack alerts. Unfortunately stacking alerts, wether they are dismissible or not, goes against [our guidance](https://design.va.gov/components/alert/#how-to-use-alerts). You can use multiple alerts if necessary, but you cannot stack them one on top of another.

**Comment by pmclaren19** [comment]
@uxgary @skylerschain do we have any designs for this yet regarding the tracked item alerts? If we do could you link those ticket to this one? If not, could we create a ticket to do this design work? Thanks!

**Comment by skylerschain** [comment]
I've scheduled a Platform Design OH session for Nov 26 (first available) to discuss this further with the Platform team.

**Comment by skylerschain** [comment]
Notes from Office Hours (Nov 26): 

To explore: A single “parent alert” with separate entry points for tracked items that are not alerts in the “What you need to do” section.

**Comment by jstrothman** [comment]
@sh0wn @salihaghaffar We have this [initial design exploration (Figma)](https://www.figma.com/design/m1Xt8XjVDjZIbliCYcCKpE/Document-status?node-id=2942-23143&t=dhYpZ2nlu0apVlNY-4) for stacked alerts. 

Could you add this issue to our list for prioritization discussion for next sprint?

**Comment by kirkacrawford-VA** [comment]
> [@kirkacrawford-VA](https://github.com/kirkacrawford-VA) this has been a known issue for a while and is something BMT 1 is taking on as part of this epic: [CST Alerts and Alignment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/119389). Although BMT 1 and 2 may collaborate on this, I think we can remove this ticket from BMT 2's backlog since BMT 1 will be taking the lead on it. Thoughts?
> 
> cc: [@mcarterA6](https://github.com/mcarterA6) [@jstrothman](https://github.com/jstrothman)

Thanks @amylai-va ! Agree with moving this outside of BMT2's backlog. Shall we also update the parent issue to https://github.com/department-of-veterans-affairs/va.gov-team/issues/119389 to show the linkage?
